src/Contracts/Helpers/ConfigRetrieverInterface.php

```diff
@@ -9,8 +9,6 @@ interface ConfigRetrieverInterface
      * @param array  $additionalConfigKeys
      *
      * @return \SocialiteProviders\Manager\Contracts\ConfigInterface
-     *
-     * @throws \SocialiteProviders\Manager\Exception\MissingConfigException
```

Comment:
Does this not throw this exc?

Reply:
No, as `\SocialiteProviders\Manager\Helpers\ConfigRetriever::fromServices` does not either.